other/concat-exports.js

```diff
@@ -0,0 +1,43 @@
+const appendToFile = require('fs').appendFileSync
+const resolve = require('path').resolve
+const htmlTagNames = require('html-tag-names')
+const svgTagNames = require('svg-tag-names')
+
+function capitalize(s) {
+  return s.slice(0, 1).toUpperCase() + s.slice(1)
+}
+
+function dashToCamelCase(s) {
+  return s.replace(/-([a-z])/g, ([, char]) => char.toUpperCase())
+}
+
+function isCapitalized(s) {
+  return !!s && s[0] === s[0].toUpperCase()
+}
+
+// node version <= 7 does not support trailing commas in function calls
+// prettier-ignore
+const jsBuiltInConstructorNames = Object.getOwnPropertyNames(global)
+  .filter(isCapitalized)
+
+function unCollide(name) {
+  const postfix = 'Tag'
+  return jsBuiltInConstructorNames.includes(name) ? `${name}${postfix}` : name
+}
+
+const elementExports = htmlTagNames
+  .concat(svgTagNames)
+  .filter((tag, index, array) => array.indexOf(tag) === index)
+  .map(tag => {
+    const validJsName = dashToCamelCase(tag)
+    const capitalName = capitalize(validJsName)
+    // add postfix if name collides with js constructors
+    const tagName = unCollide(capitalName)
+    return `export const ${tagName} = glamorous['${tag}'];`
+  }).join`\n`
+
+const toWrite = `\n${elementExports}`
```

Comment:
It would be handy to add a comment for the first line that says:

> // these exports below are generated and will be tree-shaken if you're using Webpack 2 or Rollup